PHILOSOPHY.md

```diff
@@ -0,0 +1,158 @@
+## Design goals
+
+The design goals for Tendermint (and the SDK and related libraries) are:
+
+ * Simplicity and Legibility
+ * Parallel performance, namely ability to utilize multicore architecture
+ * Ability to evolve the codebase bug-free
+ * Debuggability
```

Comment:
What do we mean by this? Tracing tooling? Code written in a way that's easy to debug?